src/main/scala/com/atomist/rug/test/gherkin/AbstractExecutableFeature.scala

```diff
@@ -45,6 +45,8 @@ abstract class AbstractExecutableFeature[W <: ScenarioWorld](
             runGiven(world, step)
           case "When " if !world.aborted =>
             runWhen(world, step)
+          case "Then " if step.getText == "the scenario aborted" =>
```

Comment:
At the very least, it feels like we need to know the difference between 'throw Error" from within the JS vs any other exception thrown by the runtime.